apps/web-giddh/src/app/subscription/subscription.component.ts

```diff
@@ -1,769 +1,372 @@
-        private changeDetection: ChangeDetectorRef,
-        private generalService: GeneralService,
-        private componentStore: SubscriptionComponentStore,
-        private store: Store<AppState>,
-        private formBuilder: FormBuilder,
-        private readonly componentStoreBuyPlan: BuyPlanComponentStore,
-        private generalActions: GeneralActions,
+    /** This will hold toggle buttons value and size */
+    public bootstrapToggleSwitch = BootstrapToggleSwitch;
+    /* Holds Mat Table Columns*/
+    public displayedColumns: string[] = ['ipaddress', 'signindate', 'signintime', 'duration', 'agent', 'action'];
+    /** Holds Active Tab Index */
+    public activeTabIndex: number = 0;
+    /** Holds Tab Name */
+    private tabName = ['auth-key', 'mobile-number', 'session', 'subscription'];
+    /** Holds True if API calling in progress in old subscription page */
+    public isSubscriptionLoading: boolean = false;
+    /** Holds subscription id */
+    public subscriptionId: string = '';
+
+    constructor(private store: Store<AppState>,
+        private toasty: ToasterService,
+        private loginService: AuthenticationService,
+        private loginAction: LoginActions,
         private router: Router,
-        private toasterService: ToasterService
-    ) {
-        this.store.dispatch(this.generalActions.openSideMenu(true));
+        private sessionAction: SessionActions,
+        private route: ActivatedRoute,
+        private breakPointObservar: BreakpointObserver,
+        private generalActions: GeneralActions,
+        private changeDetectionRef: ChangeDetectorRef,
+        private clipboardService: ClipboardService) {
+        this.contactNo$ = this.store.pipe(select(s => {
+            if (s.session.user) {
+                return s.session.user.user.contactNo;
+            }
+        }), takeUntil(this.destroyed$));
+        this.countryCode$ = this.store.pipe(select(s => {
+            if (s.session.user) {
+                return s.session.user.countryCode;
+            }
+        }), takeUntil(this.destroyed$));
+
+        this.isAddNewMobileNoInProcess$ = this.store.pipe(select(s => s.login.isAddNewMobileNoInProcess), takeUntil(this.destroyed$));
+        this.isAddNewMobileNoSuccess$ = this.store.pipe(select(s => s.login.isAddNewMobileNoSuccess), takeUntil(this.destroyed$));
+        this.isVerifyAddNewMobileNoInProcess$ = this.store.pipe(select(s => s.login.isVerifyAddNewMobileNoInProcess), takeUntil(this.destroyed$));
+        this.isVerifyAddNewMobileNoSuccess$ = this.store.pipe(select(s => s.login.isVerifyAddNewMobileNoSuccess), takeUntil(this.destroyed$));
+        this.userSessionResponse$ = this.store.pipe(select(s => s.userLoggedInSessions.Usersession), takeUntil(this.destroyed$));
+        this.isUpdateCompanyInProgress$ = this.store.pipe(select(s => s.settings.updateProfileInProgress), takeUntil(this.destroyed$));
+
+        this.authenticateTwoWay$ = this.store.pipe(select(s => {
```

Comment:
don't use single character variable

Reply:
Resolved but this is an old subscription